ecosystem/sep-0039.md

```diff
@@ -0,0 +1,155 @@
+
+## Preamble
+
+```
+SEP: 0039
+Title: Interoperability Recommendations for NFTs
+Authors: Tomer Weller <tomer@stellar.org>, Frederic Rezeau <frederic.rezeau@gmail.com>
```

Comment:
Was @tomerweller an author of this? I didn't think he was. Maybe this is a copy-paste mistake from another SEP. @Shaptic If you were wanting to not list yourself personally, you could place "SDF" here.

Reply:
Yeah this was a product of some language discussion we had offline; putting SDF is much better!